ergani/client.py

```diff
@@ -0,0 +1,228 @@
+        self.base_url = base_url
+
+    def _request(
+        self, method: str, endpoint: str, payload: Optional[Dict[str, Any]] = None
+    ) -> Optional[Response]:
+        url = f"{self.base_url}/{endpoint}"
+        response = requests.request(
+            method,
+            url,
+            json=payload,
+            auth=ErganiAuthentication(self.username, self.password, self.base_url),
+            headers={"Content-Type": "application/json"},
+        )
+
+        return self._handle_response(response, payload)
+
+    def _handle_response(
+        self, response: Response, payload: Optional[Dict[str, Any]] = None
+    ) -> Optional[Response]:
+        if response.status_code == 401:
+            error_message = extract_error_message(response)
+            raise AuthenticationError(message=error_message, response=response)
+
+        if response.status_code == 204:
+            return None
+
+        try:
+            response.raise_for_status()
+            return response
+        except:
+            error_message = extract_error_message(response)
+            raise APIError(message=error_message, response=response, payload=payload)
+
+    def submit_work_card(
+        self,
+        employer_tax_identification_number: str,
+        business_branch_number: str,
+        employee_tax_identification_number: str,
+        employee_last_name: str,
+        employee_first_name: str,
+        work_card_movement_type: str,
+        work_card_submission_date: str,
+        work_card_movement_datetime: str,
+        comments: Optional[str] = "",
+        late_declaration_justification_code: Optional[str] = "",
+    ) -> Tuple[Dict[str, Any], Optional[Response]]:
+        endpoint = "/Documents/WRKCardSE"
+
+        request_payload = {
+            "Cards": {
+                "Card": [
+                    {
+                        "f_afm_ergodoti": employer_tax_identification_number,
+                        "f_aa": business_branch_number,
+                        "f_comments": comments,
+                        "Details": {
+                            "CardDetails": [
+                                {
+                                    "f_afm": employee_tax_identification_number,
+                                    "f_eponymo": employee_last_name,
+                                    "f_onoma": employee_first_name,
+                                    "f_type": work_card_movement_type,
+                                    "f_reference_date": work_card_submission_date,
+                                    "f_date": work_card_movement_datetime,
+                                    "f_aitiologia": late_declaration_justification_code,
+                                }
+                            ]
+                        },
+                    }
+                ]
+            }
+        }
+
+        response = self._request("POST", endpoint, request_payload)
+
+        return request_payload, response
```

Comment:
We can just return `response` and get access to the request payload from `response.request`.